mobile/src/lib/cozy-helper.js

```diff
@@ -82,6 +82,27 @@ export function refreshFolder (dispatch, getState) {
   }
 }
 
+export const startRepeatedReplication = (dispatch, getState) => {
+  const options = {
+    onError: onError(dispatch, getState),
+    onComplete: refreshFolder(dispatch, getState)
+  }
+  cozy.client.offline.startRepeatedReplication('io.cozy.files', 15, options)
+}
+
+export const startFirstReplication = (dispatch, getState) => {
+  const options = {
+    onError: onError(dispatch, getState),
+    onComplete: () => {
+      dispatch(setFirstReplication(true))
```

Comment:
I don't know how, but the lib should not depend on redux.
Maybe you can pass `onError` and `onCompleete` as parameters instead of redux variable.